htdocs/product/class/html.formproduct.class.php

```diff
@@ -655,13 +655,11 @@ public function selectLotDataList($htmlname = 'batch_id', $empty = 0, $fk_produc
 		}
 
 		foreach ($productIdArray as $productId) {
-			if (array_key_exists($productId, $this->cache_lot)) {
```

Comment:
Is removing this line to avoid warning was done on purpose ?

Reply:
I seems like a mistake, I fix it

Reply:
I still don't see the protection in PR.